### PR TITLE
Adding Tilequery API example

### LINKED ISSUE #863: Add Tilequery API example
Tilequery API support has been brought to our Java SDK https://github.com/mapbox/mapbox-java/pull/879 . Would be good to have an example in the demo app.

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #863 by adding an example of Tilequery API usage.

Tests are crashing because the Java SDK dependency hasn't been updated in the Maps SDK yet, so the `MapboxTilequery` class can't be found by the demo app yet.  Creating a pr anyways so that everyone's aware of this work going on.

cc @osana @zugaldia

### CODE REVIEW

**Comment by langsmith** [comment]
Current state of this pr. Things are looking good. Queries for up to 10 building polygons which are within 50 meters of the single map click location.

ezgif com-resize

**Comment by zugaldia** [comment]
@langsmith that's amazing :tada: what do you need next, a review pass?

**Comment by langsmith** [comment]
Another option is to create two separate examples. This one and then a different example which uses Enterprise Boundaries.

**Comment by langsmith** [comment]
I think let's review and merge this. It'll be good to have a very basic version of using Tilequery. I can do any number of other Tilequery examples with(out) enterprise boundaries.

Great use cases listed at https://blog.mapbox.com/unlock-context-rich-experiences-with-the-new-tilequery-api-9af1ecf2b03f

 
screen shot 2018-12-04 at 10 20 21 am

**Comment by zugaldia** [comment]
> I think let's review and merge this. It'll be good to have a very basic version of using Tilequery. I can do any number of other Tilequery examples with(out) enterprise boundaries.

:+1: I like that approach.

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -11,12 +11,14 @@ ext {
 
             // Mapbox
             mapboxMapSdk             : '6.7.1',
-            mapboxTurf               : '4.1.1',
+            mapboxTurf               : '4.1.0',
+            mapboxServices           : '4.1.0',
```
Comment (osana) [inline gradle/dependencies.gradle]: I guess this can be 4.2.0 now

Comment (langsmith) [reply gradle/dependencies.gradle]: Bumped!

**Pull request merged**